src/lro/models.ts

```diff
@@ -2,9 +2,11 @@ import {
   OperationArguments,
   OperationSpec,
   RestResponse,
-  HttpMethods
+  HttpMethods,
+  HttpOperationResponse
 } from "@azure/core-http";
 import { PollOperationState, PollOperation } from "@azure/core-lro";
+export const LROSYM = Symbol.for("LROData");
```

Comment:
Can we have a longer name, since this is a global symbol cache?

Proposal: `packagename.identifier` e.g. `@azure/core-http.LROData`